apps/ideaspace/src/components/common/IdeaCard/IdeaCardImg.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Link from 'next/link';
+import {
+    ImgButton,
+} from './StyledIdeaCard';
+
+const IdeaCardImg = ({
+    cardId,
+    cardImg,
+}) => {
+
+    return (
+        <Link href={`/ideaspace/workshop/${cardId}`}>
+            <ImgButton>
+                {cardImg ?
+                    <img
+                        alt="idea_image"
+                        src={cardImg}
+                        style={{ width: "100%", height: "100%", borderRadius: "1rem", }}
```

Comment:
Could we add a new styled component for this image to take care of the duplicated inline styles here?